doc/HOWTO.md

```diff
@@ -67,42 +66,7 @@ heirarchy convention of `recipes/[package name]/[package version]/`
 
 ## Build the recipe ##
 
-The ideal build environment is a fresh virtual machine whose OS
-distribution and release number matches the Conda channel to which the
-new package will be released.
-
-Our Conda recipes typically do not declare build dependencies on
-generic tools such as `make`, `autotools`, `libtool` and
-`pkg-config`. You will need to install these using the OS package
-manager as required. Similarly, VCS such as `git` or `hg` are not
-typically declared.
-
-Building may be as simple as running
-
-```bash
-conda build ./recipes/[path to recipe]
-```
-
-If there are errors Conda will report the full path to the failed
-build so that you can investigate. Common reasons for build failures
-(aside from errors in the new recipe) are
-
-* The software being packaged requiring an older or newer version of a
-  compiler, build tool or library than is available in the channels
-
-* The software being packaged having a build system that fails to
-  respect `$PREFIX` during installation
-
-* The software being packaged having undocumented dependencies
-
-A successfully built package will be dropped in the output root
-directory, the default being `<CONDA_PREFIX>/conda-bld/`. This may be
-changed in the `.condarc` file or by setting the `CONDA_BLD_PATH`
-environment variable, see
-[Conda build configuration](https://conda.io/docs/user-guide/configuration/use-condarc.html#specify-conda-build-output-root-directory-root-dir)
-section of the
-[Conda User guide](https://conda.io/docs/user-guide/index.html)
-
+See doc/HOWTO.md for detailed build instructions.
```

Comment:
Refers to 'doc/HOWTO.md', even though the current path and file is 'doc/HOWTO.md' (self-reference)